### PR TITLE
Add alternate house image for specific screen sizes

### PR DESCRIPTION
**Pull request by maya**
This adds an alternate house image for specific screen sizes using an aspect-ratio media query.

TODO:
- [x] get a shorter house image from @ericadeahl 

This is what is looks like:

screen shot 2016-02-25 at 11 06 49 am

Related: https://github.com/18F/ffd-microsite/issues/1#issuecomment-186387861

### CODE REVIEW

**Comment by ericadeahl** [comment]
@maya Looks good to me!

**Comment by rogeruiz** [comment]
Looks good to me for this specific screen size @maya and @ericadeahl.

I'm seeing issues with the background on some preset devices in Chrome's browser emulation. The issues fall into two categories. **copy on top of background image** and **vertical centering of content in hero**.
- Nexus 5X Landscape ( 731 x 411 )
  - screen shot 2016-02-25 at 2 30 31 pm
- Nexus 6P Landscape ( 773 x 435 )
  - screen shot 2016-02-25 at 2 26 31 pm
- Galaxy S5 Landscape ( 640 x 360 )
  - screen shot 2016-02-25 at 2 29 26 pm
- iPhone 4 Landscape ( 480 x 320 )
  -  screen shot 2016-02-25 at 2 28 52 pm
- iPhone 5 Landscape ( 568 x 320 )
  - screen shot 2016-02-25 at 2 23 33 pm

For the **vertical centering of content in hero** issue, I think tweaking the `padding-top` from the default of `6rem` to `3rem` _or something along those lines_ we can get the content to center visually a bit better. As for the other issue **copy on top of background image**, I defer to @ericadeahl for advice on how to better handle that design. i.e. remove the image background?

**Comment by maya** [comment]
@rogeruiz thanks, we aren't vertically centering the text but adjusting the margins is a good idea.

**Comment by ericadeahl** [comment]
@rogeruiz @maya Agreed that for all these mobile landscape sizes we should remove the background image and try to vertically center that block of header text.

**Comment by rogeruiz** [comment]
Cool thanks @ericadeahl. Looks like the `min-height` might be somewhere around `435px` @maya

File: assets/styles/homepage.scss [hunk 1]
```diff
@@ -64,6 +64,10 @@ p {
   padding-top: 6rem;
   text-align: center;
 
+  @media screen and (min-aspect-ratio: 10/7) {
+    background-image: url('../images/houses-wide.svg');
+  }
+
   @media (max-height: 400px) {
```
Comment (rogeruiz) [inline assets/styles/homepage.scss]: This should be at least `435` in order to make the background disappear a bit earlier.  @maya

File: assets/styles/homepage.scss [hunk 2]
```diff
@@ -64,6 +64,10 @@ p {
...
     background-image: none;
```
Comment (rogeruiz) [inline assets/styles/homepage.scss]: after this line, a `padding-top` smaller than `6em` would have the text visually center a bit better. @maya

**Comment by maya** [comment]
@rogeruiz pushed up an update. Let me know if that works. I changed the min height from 400px to 435px and updated margins.

iPhone 5S
screen shot 2016-02-25 at 12 49 05 pm

iPhone 6
screen shot 2016-02-25 at 12 40 06 pm

Galaxy S6
screen shot 2016-02-25 at 1 09 16 pm

iPad mini
screen shot 2016-02-25 at 12 47 12 pm

**Comment by rogeruiz** [comment]
Looks good to me! @ericadeahl can you take a look at @maya's screenshots and let me know if this is good design-wise? Thanks!

**Comment by ericadeahl** [comment]
@maya @rogeruiz Looks good! The iPhone 6 one is still a little awkward - it would look better with larger text and less empty space below - but it's not a big deal. We can leave it alone if changing it would mess up the others. Otherwise :+1:

**Comment by maya** [comment]
@ericadeahl ya there'll probably be some edge cases like that one, but I think going smaller text and margins will allow us to cover our bases better.

**Comment by ericadeahl** [comment]
@maya Sounds good!

**Pull request merged**